### PR TITLE
{bio}[GCC/10.3.0] StringTie v2.1.7

### PR DESCRIPTION
**Pull request by SethosII**
(created using `eb --new-pr`)

I used the GitHub source because the most recent release isn't available on their homepage.

### CODE REVIEW

**Comment by SethosII** [comment]
Test report by @SethosII
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
ribnode012 - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2640 v2 @ 2.00GHz, Python 3.8.10
See https://gist.github.com/942d893751f9621790c51cbe843e8bef for a full test report.

File: easybuild/easyconfigs/s/StringTie/StringTie-2.1.7-GCC-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+name = 'StringTie'
+version = '2.1.7'
+
+homepage = 'https://ccb.jhu.edu/software/stringtie/'
+description = 'StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts'
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/gpertea/%(namelower)s/releases/download/v%(version)s']
```
Comment (verdurin) [inline easybuild/easyconfigs/s/StringTie/StringTie-2.1.7-GCC-10.3.0.eb:16]: @SethosII isn't this the right version?: 
`https://ccb.jhu.edu/software/stringtie/dl/stringtie-2.1.7.tar.gz` ?

Comment (SethosII) [reply easybuild/easyconfigs/s/StringTie/StringTie-2.1.7-GCC-10.3.0.eb:16]: @verdurin Ah ok, so they haven't updated it on the website (http://ccb.jhu.edu/software/stringtie/ has 2.1.6 as the latest and the changelog there is only up to 2.1.5). Both files are identical (I checked their SHA256). Should I change it to the url of the homepage?

Comment (verdurin) [reply easybuild/easyconfigs/s/StringTie/StringTie-2.1.7-GCC-10.3.0.eb:16]: @SethosII I don't have strong feelings. If they make proper releases on Github, that's fine. 
Whichever is the most reliable.

Comment (SethosII) [reply easybuild/easyconfigs/s/StringTie/StringTie-2.1.7-GCC-10.3.0.eb:16]: @verdurin Ok, then I would use GitHub and hope that they have a higher availability ;)

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node379.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/55ecc789635775392fe9032624d6acb8 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @SethosII !

**Pull request merged**